### PR TITLE
 [ArchWall] Do not copy shape

### PR DESCRIPTION
**Pull request by paullee0**
Discussions- 
https://forum.freecadweb.org/viewtopic.php?style=1&t=62968&p=540585 realthunder@2348994
https://github.com/realthunder/FreeCAD/commit/234899409f847b933c414a25deb91d9d2cb7bc88

Thank you for creating a pull request to contribute to FreeCAD! To ease integration, we ask you to conform to the following items. Pull requests which don't satisfy all the items below might be rejected. If you are in doubt with any of the items below, don't hesitate to ask for help in the [FreeCAD forum](https://forum.freecadweb.org/viewforum.php?f=10)!

- [x]  Your pull request is confined strictly to a single module. That is, all the files changed by your pull request are either in `App`, `Base`, `Gui` or one of the `Mod` subfolders. If you need to make changes in several locations, make several pull requests and wait for the first one to be merged before submitting the next ones
- [ ]  In case your pull request does more than just fixing small bugs, make sure you discussed your ideas with other developers on the FreeCAD forum
- [x]  Your branch is [rebased](https://git-scm.com/docs/git-rebase) on latest master `git pull --rebase upstream master`
- [ ]  All FreeCAD unit tests are confirmed to pass by running `./bin/FreeCAD --run-test 0`
- [x]  All commit messages are [well-written](https://chris.beams.io/posts/git-commit/) ex: `Fixes typo in Draft Move command text`
- [x]  Your pull request is well written and has a good description, and its title starts with the module name, ex: `Draft: Fixed typos`
- [ ]  Commit messages include `issue #<id>` or `fixes #<id>` where `<id>` is the [FreeCAD bug tracker issue number](https://freecadweb.org/wiki/tracker#GitHub_and_MantisBT) in case a particular commit solves or is related to an existing issue on the tracker. Ex: `Draft: fix typos - fixes #0004805`

And please remember to update the Wiki with the features added or changed once this PR is merged.  
**Note**: If you don't have wiki access, then please mention your contribution on the [0.20 Changelog Forum Thread](https://forum.freecadweb.org/viewtopic.php?f=10&t=56135).

---

### CODE REVIEW

**Comment by paullee0** [comment]
Why are there old commits mixed in e.g. that one on 2.1.2021 !?

Totally confused myself!  See if anyone can help?  Thanks

**Comment by chennes** [comment]
Did you have an old stash that accidentally got added? Anyway, you should be able to remove them with an interactive rebase and force push. Or we can just cherry-pick that last commit and ignore the other two.

File: ArchSketchObject.py [hunk 1]
```diff
@@ -0,0 +1,1144 @@
+#***************************************************************************	
+#*                                                                         *	
+#*   Copyright (c) 2018 - 2020                                             *	
+#*   Paul Lee <paullee0@gmail.com>                                         *
```
Comment (luzpaz) [inline ArchSketchObject.py:4]: Please merge L4 in to L3 like so: 
`#*   Copyright (c) 2018-2020 Paul Lee <paullee0@gmail.com>                   *`

**Review by luzpaz — CHANGES_REQUESTED** [review]

**Comment by paullee0** [comment]
> Did you have an old stash that accidentally got added? Anyway, you should be able to remove them with an interactive rebase and force push. Or we can just cherry-pick that last commit and ignore the other three.

Thanks !  Basically, every time I make reference to this post to do git, and know almost nothing more :- 
https://forum.freecadweb.org/viewtopic.php?f=10&t=23204&start=10#p197095

I should have screwed up the local copy few weeks before during last PR.  Then, I remove the whole FC folders and start over :-
-  git clone https://github.com/myGitHubId/FreeCAD.git .
- git remote add upstream https://github.com/FreeCAD/FreeCAD.git
Catch up my newBranch with upstream/master (the "real" master):
-  git checkout master
-  git fetch upstream
-  git merge upstream/master
Develop:
-  git branch newBranchName
-  git checkout newBranchName
-  code, commit
-  (forget if did a  - git rebase master or not)
- git push --set-upstream origin ArchWall_realthunder_fix 
  ( i.e. [ArchWall] Do not copy shape  )

Not sure how I can delete the first 3 commits here...?  Thanks again !

File: ArchSketchObject.py (hunk 1 above)
Comment (paullee0) [reply ArchSketchObject.py:4]: Thanks :)

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_5222">pipeline status</a> for feature branch [PR_5222](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_5222). Pipeline [423704155](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/423704155) was triggered at [f620489](https://github.com/FreeCAD/FreeCAD/pull/5222/commits/f6204896e4fd98ec570180f8091439412d6fb1b8). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by chennes** [comment]
Is my understanding correct, that the only commit here that actually matters is f6204896e4fd98ec570180f8091439412d6fb1b8?

**Comment by paullee0** [comment]
Trying to follow to fix :
https://stackoverflow.com/questions/36168839/how-to-remove-commits-from-a-pull-request

- seems delete the ArchSketchObject.py : OK
- did i also delete the commit to ArchWall.py which should be included in this PR?

Sorry for the confusion

**Comment by chennes** [comment]
I think at this point it's best to stop trying to fix this PR: I've successfully cherry-picked the changes to ArchWall.py in f620489 (3 lines of code -- cherry-picking is where I pull out just the individual commit, and ignore the other stuff). Is there more that's supposed to be done? I seem to remember at some point there were more like 6-10 lines.

**Comment by chennes** [comment]
I pushed that one commit, anyway, it's now: ea25f69f03a2dfb8b0a424186e320a2d20c7f657

If that's all that needed to go in, you can close this PR. If there's more, it might actually be easier for you to still close this one, delete your old branch, update your local copy, and make a new branch for it.

**Comment by paullee0** [comment]
> I pushed that one commit, anyway, it's now: [ea25f69](https://github.com/FreeCAD/FreeCAD/commit/ea25f69f03a2dfb8b0a424186e320a2d20c7f657)
> 
> If that's all that needed to go in, you can close this PR. If there's more, it might actually be easier for you to still close this one, delete your old branch, update your local copy, and make a new branch for it.

Thanks!  That's it !

Sorry for disturbing everyone.  Would follow to fix my local copy.

**Pull request closed without merging**